### PR TITLE
styles and NWS gages

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

File: scripts/fetch/fetch.nws_data.R [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      
+      return_list <- GET(url.site)
+      returnedDoc <- content(return_list,encoding = "UTF-8")
+      nws.site <- xml_root(returnedDoc)
+      sigstages <- xml_find_all(nws.site, "sigstages")
+      
+      if(length(sigstages) > 0){
+  
+        sites$flood.stage[which(sites$NWS %in% i)] <- as.numeric(xml_text(xml_find_all(sigstages, "flood")))
+        sites$flood.stage.units[which(sites$NWS %in% i)] <- xml_attr(xml_find_all(sigstages, "flood"),"units")
```
Comment (dblodgett-usgs) [inline scripts/fetch/fetch.nws_data.R:32]: Funny for i to be used for something other than an index?

File: scripts/fetch/fetch.nws_data.R [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      }
+    }
+    sites <- sites %>% filter(!is.na(flood.stage))
+    location <- viz[['location']]
+    saveRDS(sites, file=location)
+  } else {
+    return(NULL)
+  }
+  
+}
```
Comment (dblodgett-usgs) [inline scripts/fetch/fetch.nws_data.R:42]: Looks good!

File: scripts/process/process.storm_sites.R [hunk 3]
```diff
@@ -33,8 +40,10 @@ process.getNWISdata <- function(viz = as.viz('gage-data')){
   checkRequired(viz, required)
   depends <- readDepends(viz)
   siteInfo <- depends[['storm-sites']]
-  sites_active <- dplyr::filter(siteInfo@data, class == 'active-gage')$id
+  sites_active <- dplyr::filter(siteInfo@data, class == 'nwis-dot')$id
   sites_active <- gsub(pattern = "nwis-", replacement = "", x = sites_active)
+  # now, use the SORTED dependency "sites" to get these in latitude order
+  sites_active <- depends$sites$site_no[depends$sites$site_no %in% sites_active]
```
Comment (dblodgett-usgs) [inline scripts/process/process.storm_sites.R:46]: This isn't quite right. I've got a fix coming.

File: viz.yaml [hunk 4]
```diff
@@ -158,13 +158,28 @@ fetch:
...
+    depends: ['sites', 'nws-conversion']
+  -
     id: storm-area-filter
     location: cache/storm-area-filter.rds
     reader: rds
     fetcher: storm_area_filter
     lats: [24.5, 30.9, 32.7, 27, 24.5, 24.5]
     lons: [-83.4, -84, -80.5, -78.9, -79.6, -83.4]
-    height: 10
+    height: 10 # we should instead depend on view-limits and get all of this out of that target
```
Comment (wdwatkins) [inline viz.yaml]: this isn't actually being used, it is just leftover copypasta.  `lats` and `lons` sets everything.

File: scripts/fetch/fetch.nws_data.R (hunk 1 above)
Comment (jordansread) [reply scripts/fetch/fetch.nws_data.R:32]: yeah, I don't like this style. It is someone elses code from Harvey

File: viz.yaml (hunk 4 above)
Comment (jordansread) [reply viz.yaml]: I will delete then

**Comment by dblodgett-usgs** [comment]
:shipit: AFAIK

File: viz.yaml (hunk 4 above)
Comment (jordansread) [reply viz.yaml]: looks like the proj.string is used in the storm-counties process, but I will fix that

**Pull request merged**